turbinia/workers/plaso.py

```diff
@@ -51,6 +52,21 @@ def run(self, evidence, result):
         '--partition all --vss_stores all').split()
     if config.DEBUG_TASKS:
       cmd.append('-d')
+
+    if isinstance(evidence, BitlockerDisk):
+      if evidence.recovery_key:
+        cmd.extend([
+            '--credential', 'recovery_password:{0:s}'.format(
```

Comment:
Huh, weird that YAPF formatted it like this.  It seems like since this already multiple elements, that it would be a little more readable if the second element was alone on the second line.